changelogs/changelog.yaml

```diff
@@ -274,3 +274,9 @@ releases:
         name: snmp_manager
         namespace: ''
     release_date: '2024-11-29'
+  3.7.0:
+    changes:
+      minor_changes:
+        - Host module is enhanced to support check and diff mode.
+        - Volume module is enhanced to support check and diff mode.
```

Comment:
update bug fix here as well, applicable to all such instances.

Reply:
done